packages/gatsby-cli/src/reporter/redux/actions.ts

```diff
@@ -1,44 +1,21 @@
 import { bindActionCreators } from "redux"
 import { Dispatch } from "redux"
 import { dispatch } from "./"
-import {
-  createLog as internalCreateLog,
-  createPendingActivity as internalCreatePendingActivity,
-  setStatus as internalSetStatus,
-  startActivity as internalStartActivity,
-  endActivity as internalEndActivity,
-  updateActivity as internalUpdateActivity,
-  setActivityErrored as internalSetActivityErrored,
-  setActivityStatusText as internalSetActivityStatusText,
-  setActivityTotal as internalSetActivityTotal,
-  activityTick as internalActivityTick,
-} from "./internal-actions"
 
-const actions = {
-  createLog: internalCreateLog,
-  createPendingActivity: internalCreatePendingActivity,
-  setStatus: internalSetStatus,
-  startActivity: internalStartActivity,
-  endActivity: internalEndActivity,
-  updateActivity: internalUpdateActivity,
-  setActivityErrored: internalSetActivityErrored,
-  setActivityStatusText: internalSetActivityStatusText,
-  setActivityTotal: internalSetActivityTotal,
-  activityTick: internalActivityTick,
-}
+import * as actions from "./internal-actions"
```

Comment:
👍

Reply:
btw, changes in this file I can revert to avoid thrashing, I thought I would need to mess with things here, but in the end I didn't. (tho I do think it's a bit neater/shorter with this change :) )

Reply:
I also like the change, keep it :)